extensions/big-data-cluster/src/kubectl/kubectl.ts

```diff
@@ -72,17 +75,17 @@ async function checkForKubectlInternal(context: Context, errorMessageMode: Check
     const bin = getToolPath(context.host, context.shell, binName);
 
     const contextMessage = getCheckKubectlContextMessage(errorMessageMode);
-    const inferFailedMessage = `Could not find "${binName}" binary.${contextMessage}`;
-    const configuredFileMissingMessage = `${bin} is not installed. ${contextMessage}`;
+    const inferFailedMessage = localize('binaryNotFound', 'Could not find {0} binary. {1}', binName, contextMessage);
+    const configuredFileMissingMessage = localize('binaryNotInstalled', '${0} is not installed. ${1}', bin, contextMessage);
```

Comment:
same here, remove $

Reply:
Whoops - will fix all of the many issues with this you spotted. Thanks!